docs/synapse/userguides/storm_ref_filter.rstorm

```diff
@@ -793,247 +917,449 @@ know the exact order or values of the array itself.
 
 **Examples:**
 
-- From an initial set of domains, filter results to only those domains that resolve to an IP address
-  that Trend Micro associates with the Pawn Storm threat group (i.e., an IP address tagged
-  ``#rep.trend.pawnstorm``):
+*Filter the current working set of FQDNs (inet:fqdn nodes) to only FQDNs that have resolved to an IPv4 address that Trend Micro associates with Pawn Storm (i.e., an IP address tagged #rep.trend.pawnstorm):*
 
-.. storm-pre:: inet:fqdn +{ -> inet:dns:a:fqdn :ipv4 -> inet:ipv4 +#rep.trend.pawnstorm }
+.. storm-pre:: inet:fqdn +{ -> inet:dns:a -> inet:ipv4 +#rep.trend.pawnstorm }
 ::
+  
+  <inet:fqdn> +{ -> inet:dns:a -> inet:ipv4 +#rep.trend.pawnstorm }
+
+The subquery filter above takes the inbound ``inet:fqdn`` nodes and (within the subquery):
 
-    <inet:fqdn> +{ -> inet:dns:a:fqdn :ipv4 -> inet:ipv4 +#rep.trend.pawnstorm }
+- pivots to the associated DNS A records (``inet:dns:a`` nodes);
+- pivots to the asssociated IPv4 addresses (``inet:ipv4`` nodes);
+- checks the IPv4 for the presence of a ``#rep.trend.pawnstorm`` tag.
 
+The subquery filter returns only those ``inet:fqdn`` nodes where, if you performed the operations
+within the subquery, **would** (based on the inclusive filter) result in an ``inet:ipv4`` node with a
+``#rep.trend.pawnstorm`` tag.
 
-- From an initial set of IPv4 addresses, filter results to only those IPv4s registered to an Autonomous
-  System (AS) whose name starts with "makonix":
 
-.. storm-pre:: inet:ipv4 +{ :asn -> inet:asn +:name^="makonix" }
+*Filter the current working set of IPv4 addresses (inet:ipv4 nodes) to exclude any IPv4 associated with an Autonomous System (AS) whose name starts with "makonix":*
+
+.. storm-pre:: inet:ipv4 -{ :asn -> inet:asn +:name ^= makonix }
 ::
+  
+  <inet:ipv4> -{ :asn -> inet:asn +:name ^= makonix }
+
+The subquery filter above takes the inbound ``inet:ipv4`` nodes and (within the subquery):
 
-    <inet:ipv4> +{ :asn -> inet:asn +:name^="makonix" }
+- pivots to the associated ``inet:asn`` nodes; and
+- checks the ``inet:asn`` nodes for a ``:name`` value that starts with "makonix".
 
+The subquery filter returns only those ``inet:ipv4`` nodes where, if you performed the operations
+within the subquery, **would not** (based on the exclusive filter) result in an ``inet:asn`` node
+with a ``:name`` value starting with "makonix".
+
+.. TIP::
+  
+  See :ref:`embed_prop_syntax` for an alternative way to perform this query.
 
-- From an initial set of ``file:bytes`` nodes, filter results to only those that are detected as
-  malicious by ten (10) or more antivirus / malscanner vendors (i.e., files that are associated with
-  10 or more ``it:av:filehit`` nodes):
+
+*Filter the current working set of files (file:bytes nodes) to include only files that are detected as malicious by ten (10) or more antivirus / malscanner vendors (i.e., files that are associated with   10 or more it:av:filehit nodes):*
```

Comment:
```suggestion
*Filter the current working set of files (file:bytes nodes) to include only files that are detected as malicious by ten (10) or more antivirus / malscanner vendors (i.e., files that are associated with 10 or more it:av:filehit nodes):*
```

Reply:
Also it:av:filehit is deprecated, we should probably update this example

Reply:
Blergh. Updated.